addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -25,12 +25,8 @@ registerModel({
     name: 'Chatter',
     identifyingFields: ['id'],
     lifecycleHooks: {
-        _created() {
-            this._attachmentsLoaderTimeout = undefined;
-            this._isPreparingAttachmentsLoading = undefined;
-        },
         _willDelete() {
-            this._stopAttachmentsLoading();
+            this.env.browser.clearTimeout(this.attachmentsLoaderTimeout);
```

Comment:
Any specific reason to no longer clear the fields on delete? Just because it is kinda useless?

Reply:
It's related to last paragraph of this comment:
https://github.com/odoo/odoo/pull/82698#discussion_r784748266

to sum up, `.update()` from `_willDelete` triggers a nested update cycle during teardown of tests, and so checks in this nested update cycle make a crash because state is inconsistent.

I have removed this useless nested cycle to fix the issue